apps/base-docs/base-learn/docs/introduction-to-solidity/introduction-to-solidity-overview.md

```diff
@@ -27,7 +27,7 @@ By the end of this module, you should be able to:
   - Relate an overview of the history (and pace of change) of Solidity and its strengths and weaknesses
   - Deploy and test the Storage.sol demo contract in Remix
 - **Contracts and Basic Functions**
-  - Construct a simple ""Hello World"" contract
+  - Construct a simple ""Hello World" contract
```

Comment:
This causes there to be two open quotes and only one close quote

Reply:
@wbnns Reverted quotes